tox.ini

```diff
@@ -40,3 +40,10 @@ exclude =
 source = stripe
 omit =
     stripe/six.py
+
+[pytest]
+# This is only necessary for Python 3.3, as the `mock` module in the 3.3 stdlib
+# is not agnostic when asserting that a method was called with positional vs.
+# keyword arguments.
+# Once we drop support for Python 3.3, we can safely remove this line.
+mock_use_standalone_module = true
```

Comment:
By default, pytest-mock will use the `mock` module that comes with the stdlib in Python 3. However, as the comment explains, in 3.3 the `assert_called_with` method is not agnostic when checking for positional vs. keyword arguments, which was causing our `assert_requested` method to fail for some tests depending on how `api_requestor.request` was called.

To fix this, I added the `mock` pypi module to setup.py for Python<=3.3 and the configuration line above.

I'd rather drop support for 3.3 entirely though and get rid of this workaround.

Reply:
Thanks for the explanation and leaving the comment in here (the code).